tagstudio/src/qt/ui/settings_ui.py

```diff
@@ -0,0 +1,330 @@
+# -*- coding: utf-8 -*-
```

Comment:
add this file into `pyproject.toml` -> `ruff.tool.exclude` beside the other autogenerated files